tests/components/unifi/test_sensor.py

```diff
@@ -44,37 +50,37 @@ async def test_platform_manually_configured(hass):
     """Test that we do not discover anything or try to set up a controller."""
     assert (
         await async_setup_component(
-            hass, sensor.DOMAIN, {sensor.DOMAIN: {"platform": "unifi"}}
+            hass, SENSOR_DOMAIN, {SENSOR_DOMAIN: {"platform": UNIFI_DOMAIN}}
         )
         is True
     )
-    assert unifi.DOMAIN not in hass.data
+    assert UNIFI_DOMAIN not in hass.data
 
 
 async def test_no_clients(hass):
     """Test the update_clients function when no clients are found."""
     controller = await setup_unifi_integration(
-        hass, options={unifi.const.CONF_ALLOW_BANDWIDTH_SENSORS: True},
+        hass, options={CONF_ALLOW_BANDWIDTH_SENSORS: True},
     )
 
     assert len(controller.mock_requests) == 4
-    assert len(hass.states.async_entity_ids("sensor")) == 0
+    assert len(hass.states.async_entity_ids(SENSOR_DOMAIN)) == 0
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)